### PR TITLE
Improve HTTP errors page based on feedback

### PR DESCRIPTION
**Pull request by ghost**
An attempt to organize the HTTP errors doc based on this user feedback: https://issues.newrelic.com/browse/NR-97487

### CODE REVIEW

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/network-pages/http-errors-network-failure-analysis.mdx [hunk 1]
```diff
@@ -178,11 +176,11 @@ Error profiles provide visual details about significant differences in the frequ
 
 This helps you take more of the guesswork out of resolving your mobile application's HTTP errors. You can more easily determine if you safely ignore the error, or if you should attempt to resolve the error with a new deployment, code change, customer communication or other actions.
 
-## View more details about a specific error [#error-summary]
+## Drill down into a single error [#error-summary]
 
 To view details about an error or failure, select the **Request URL** link to be directed to the **Error summary** page. From the **Error summary** page, you can view the version information, request attributes, and **Response** body, as well as get a breakdown of error types for the request URL.
 
-## View and share error data with query builder [#insights]
+## Query and shar error data [#insights]
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/network-pages/http-errors-network-failure-analysis.mdx:183]:
```suggestion
## Query and share error data [#insights]
```

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/network-pages/http-errors-network-failure-analysis.mdx [hunk 2]
```diff
@@ -197,31 +195,7 @@ To dig deeper into the error data, [query your data](/docs/query-your-data/explo
...
-1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Network > Errors**.
-2. To change the view to errors or failures, select the **Sort by** option.
-3. To hide low-usage hosts, select the **Hide &lt; 1% throughput** option.
-4. To limit information to a specific [version](/docs/mobile-monitoring-ui/versions) of your app, or to change the time period, select your choice from the **Versions** menu or the [**time picker**](/docs/site/timepicker-setting-time-periods-to-view-data) below the menu bar.
-5. To view details for a specific host, HTTP status error, or network failure, select its name.
-
-Use any of our standard [user interface functions](/docs/accounts-partnerships/education/getting-started-new-relic/new-relic-user-interface) to drill down into detailed information.
-
-## Error trace details [#trace]
+## Get trace details  [#trace]
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/network-pages/http-errors-network-failure-analysis.mdx:198]:
```suggestion
## Get trace details [#trace]
```

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/network-pages/http-errors-network-failure-analysis.mdx [hunk 3]
```diff
@@ -178,11 +176,11 @@ Error profiles provide visual details about significant differences in the frequ
 
 This helps you take more of the guesswork out of resolving your mobile application's HTTP errors. You can more easily determine if you safely ignore the error, or if you should attempt to resolve the error with a new deployment, code change, customer communication or other actions.
 
-## View more details about a specific error [#error-summary]
+## Drill down into a single error [#error-summary]
```
Comment (stumblefiend) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/network-pages/http-errors-network-failure-analysis.mdx:179]: "drill down" seems like an idiom or phrasal verb that might not translate well to other languages or might not be understood by non-native english speakers.

Comment (ally-sassman) [reply src/content/docs/mobile-monitoring/mobile-monitoring-ui/network-pages/http-errors-network-failure-analysis.mdx:179]: I need to get this PR out and haven't heard back from the translations team yet, so I'll merge, but if they object I'll do a find/replace of any of my mentions of "drill down". Thanks for raising a potential issue with this phrase - I've always been curious if other people use it.

**Pull request merged**